client/src/api/client.js

```diff
@@ -1,5 +1,5 @@
-const fetch = global.fetch;
-const Headers = global.Headers;
+const { fetch } = global;
+const { Headers } = global;
```

Comment:
I’m a bit surprised by this, I thought destructuring on the `window` object wasn’t possible. If this does work, it’d be nicer to:

```suggestion
const { fetch, Headers } = global;
```

Reply:
Tested the page explorer and it worked for me.